packages/opentelemetry-instrumentation-http/src/utils.ts

```diff
@@ -177,6 +177,66 @@ export const setSpanWithError = (
   span.setStatus(status);
 };
 
+/**
+ * Adds attributes for request content-length and content-encoding HTTP headers
+ * @param { IncomingMessage } Request object whose headers will be analyzed
+ * @param { Attributes } Attributes object to be modified
+ */
+export const setRequestContentLengthAttribute = (
```

Comment:
What is the benefit of these functions?
It seems they are only called from tests but not from the plugin code itself.

Reply:
They are called just under by `getOutgoingRequestAttributesOnResponse` and `getIncomingRequestAttributes`